modules/virt-about-vms-and-boot-sources.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-creating-and-using-boot-sources.adoc
+// * virt/vm_templates/virt-creating-vm-template.adoc
+
+[id="virt-about-vms-and-boot-sources_{context}"]
+
+= About virtual machines and boot sources
+
+Virtual machines consist of a virtual machine definition and one or more disks, backed by datavolumes.
+
+Virtual machine templates enable you to create virtual machines using preset virtual machine definitions.
+
+Unlike a virtual machine template, a boot source is a fully configured virtual machine image that includes configured drivers. Each virtual machine template contains a virtual machine definition with a pointer to a boot source. For some templates, a boot source is automatically provided. If not, then the boot source must be prepared by an administrator.
```

Comment:
Every virtual machine template requires a boot source, which is a fully configured virtual machine disk image that includes i.e. configured drivers. Each virtual machine template contains a virtual machine definition with a pointer to a boot source with a well defined name and namespace. For some templates, a boot source is automatically provided. If not, then the boot source must be prepared by an administrator.

Without an automatic or administrator provided boot source, every user of a virtual machine template has to provide/bring it's own boot source.
The presence of an automatic or administrator provided boot source eliminates the need from the user to provide the boot source, streamlining the creation of new ready-to-use virtual machines from virtual machine templates.

Reply:
Thanks, Fabian. I have condensed this to the following using some of your wording:

_Every virtual machine template requires a boot source, which is a fully configured virtual machine disk image that includes configured drivers. Each virtual machine template contains a virtual machine definition with a pointer to the boot source with a predefined name and namespace. For some templates, a boot source is automatically provided. If it is not provided, then a custom boot source must be prepared by an administrator._

The second paragraph mostly restates the first, and I think there's enough in the first to speak to the benefits of using boot sources.